includes/abstracts/llms.abstract.notification.view.php

```diff
@@ -608,7 +609,13 @@ private function get_merged_string( $string ) {
 
 				}
 
-				$string = str_replace( $code, $this->$func( $code ), $string );
+				$string = preg_replace_callback(
+					'#' . preg_quote( $code, '#' ) . '#',
```

Comment:
Why use '#' delimiters instead of the traditional '/'?

Reply:
Actually for some reason I always use `#`.
But you made me think that in this case, since `#` are escaped by default, I don't need to pass the delimiter param, I guess...